content/api-reference/software-management/toc.yml

```
@@ -0,0 +1,10 @@
+- name: Access
```

Can you update this name to "Access control"? It's the naming convention that we follow for ACL topics.
```suggestion
- name: Access control
```